lib/qbo_api/connection.rb

```diff
@@ -51,6 +51,7 @@ def raw_request(method, conn:, path:, payload: nil, params: nil)
         when :post, :put
           req.url path
           req.body = payload.to_json
+          req.headers = headers if headers
```

Comment:
Let's move this line to before the start of the case statement so that headers can be set for a `get` call too.